apple/RNCWebView.m

```diff
@@ -431,6 +438,16 @@ - (void)setAllowsBackForwardNavigationGestures:(BOOL)allowsBackForwardNavigation
   _webView.allowsBackForwardNavigationGestures = _allowsBackForwardNavigationGestures;
 }
 
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130300 || \
```

Comment:
can we restrict it to debug builds?

Reply:
As far as I know, it's not as easy as adding `#if RCT_DEBUG` here :| 

Ideas:
* publish two versions of this and switch between them at compile time
* publish single version with `setReleaseFlag()` which permanently disables debugging at runtime once called (but caller has to win the race and be first).

Reply:
or add this functionality in a dev-mode patch instead of publishing